aiocouch/database.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, couchdb, id):
         await self._delete()
 
     async def docs(self, ids=None, create=False, prefix=None, **params):
+        if ids is not None and not ids:
+            return
```

Comment:
I'm inclined to say that passing an empty list for ids should raise a ValueError instead.

Reply:
Ok, but in that case I will not be able to use the following code:
```py
with db.update_docs([]) as bulk:
    bulk._docs.append(a_document)
    bulk._docs.append(another_document)
```
It's sad for me, because this code is useful in the situation where I know a bunch of documents ids, I have them, and I want to save them, all at once.

Reply:
That's what `db.create_docs()` was introduced for recently.